packages/marshal/src/marshal.js

```diff
@@ -402,6 +403,10 @@ export function makeMarshal(
 
           default: {
             assert(
+              // @ts-ignore This value indeed violates the current types. We
```

Comment:
nit: prefer [`@ts-expect-error`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments) in this case. If a future refactor breaks the exhaustiveness of the `switch`, we should catch it with a type system error here.

Reply:
Nice. I didn't know about that one. Done.

Reply:
@mhofman help please. Having changed it to `@ts-expect-error` At https://github.com/Agoric/agoric-sdk/pull/3821/checks?check_run_id=3626466228 I now get

> Error: ../marshal/src/marshal.js(406,15): error TS2578: Unused '@ts-expect-error' directive.

Reply:
@mhofman Some further info. In packages/marshal when I do a `yarn lint` I get the same error that CI complains about. When I remove the comment entirely, it passes locally but my IDE underlines it in red:

![image](https://user-images.githubusercontent.com/273868/133692696-51342702-7bfc-42f6-8d01-2ea5f988670e.png)

FWIW, the complain in the IDE popup is correct, so it is the `yarn lint` behavior that seems more mysterious.

Attn also @michaelfig

Reply:
Since this problem is independent of the point of this PR, I'm going to revert this to `@ts-ignore` but with a link to https://github.com/Agoric/agoric-sdk/issues/3840 that links back to this PR conversation. Then I'll merge this and we can worry about the typing issue separately.

Attn also @dckc

Reply:
Done